app/server/src/server/server.ts

```diff
@@ -22,16 +23,24 @@ app.use(compression({ level: 9 })); // Use best compression
 const rootDir = path.resolve(path.join(__dirname, "../.."));
 
 // Get command line args
-const { configPath } = require("./args");
+const options = processArgs();
 
 // Global config
-const configReader = new ConfigReader(configPath);
+const pathResolved = path.resolve(options.path);
+const configReader = new ConfigReader(pathResolved, options.overrides);
+
+console.log(`Reading config from ${pathResolved} (${options.path})`);
```

Comment:
Not sure if this console log was intended just for debug or to be permanent (same for next few lines too), but probably don't need to log both path.resolved and options.path?

Reply:
Sure, that was just copying the behaviour from the previous version (or trying to at least). I think printing these out is generally not bad though as it makes digging in to debug much easier

Reply:
The thing that the above one does do here is prints ("Reading config from /full/path/to/config.yml (../../config.yml)") where the user has given something fairly inventive.  That should not happen very much in reality